github/event.go

```diff
@@ -54,6 +54,8 @@ func (e *Event) ParsePayload() (payload interface{}, err error) {
 		payload = &DeploymentStatusEvent{}
 	case "DiscussionEvent":
 		payload = &DiscussionEvent{}
+	case "DiscussionCommentEvent":
```

Comment:
Since "DiscussionCommentEvent" is lexicographically less than "DiscussionEvent", let's please move these two lines before line 55 above.